test/testimportproject.cpp

```diff
@@ -141,6 +141,25 @@ class TestImportProject : public TestFixture {
         ASSERT_EQUALS(0, importer.fileSettings.size());
     }
 
+    void importCompileCommands5() const {
+        const char json[] =
+            "[{"
+            "\"directory\": \"C:/Users/dan/git/build-test-cppcheck-Desktop_Qt_5_15_0_MSVC2019_64bit-Debug\","
+            "\"command\": \"C:\\\\PROGRA~2\\\\MICROS~1\\\\2019\\\\COMMUN~1\\\\VC\\\\Tools\\\\MSVC\\\\1427~1.291\\\\bin\\\\HostX64\\\\x64\\\\cl.exe  /nologo /TP -IC:\\\\Users\\\\dan\\\\git\\\\test-cppcheck\\\\mylib\\\\src /DWIN32 /D_WINDOWS /GR /EHsc /Zi /Ob0 /Od /RTC1 -MDd -std:c++17 /Fomylib\\\\CMakeFiles\\\\mylib.dir\\\\src\\\\foobar\\\\mylib.cpp.obj /FdTARGET_COMPILE_PDB /FS -c C:\\\\Users\\\\dan\\\\git\\\\test-cppcheck\\\\mylib\\\\src\\\\foobar\\\\mylib.cpp\","
```

Comment:
It seems to me it would be interesting with a include path with a space also. do you know if we have that somewhere? maybe that would just generate some "~" path then it's not very interesting.

Reply:
I agree I will add another test for that, folder-names with spaces are quite common on windows

Reply:
And I found an issue with folders and spaces :-)
added a fix for that